articles/active-directory-b2c/supported-azure-ad-features.md

```diff
@@ -24,8 +24,16 @@ An Azure Active Directory B2C (Azure AD B2C) tenant is different than an Azure A
-| [Premium P2](https://azure.microsoft.com/pricing/details/active-directory/) | Fully supported for Azure AD premium P2 features. For example, [Identity Protection](../active-directory/identity-protection/overview-identity-protection.md), and [Identity Governance](../active-directory/governance/identity-governance-overview.md).  | Azure AD B2C uses [Azure AD B2C Premium P2 license](https://azure.microsoft.com/pricing/details/active-directory/external-identities/), which is different from Azure AD premium P2. A subset of Azure AD Identity Protection features is supported with [consumer accounts](user-overview.md#consumer-user). Learn how to [Investigate risk with Identity Protection](identity-protection-investigate-risk.md) and configure Azure AD B2C [Conditional Access](conditional-access-user-flow.md). |
+| [Administrative Units](../active-directory/roles/administrative-units.md) | Fully supported with Azure AD premium P1 licensing | Azure AD B2C tenants do not support Administrative Units. |
+| [Dynamic Groups](../active-directory/enterprise-users/groups-dynamic-membership.md) | Fully supported with Azure AD premium P1 licensing | Azure AD B2C tenants do not support Dynamic Groups. |
+| [Privileged Identity Management](../active-directory/privileged-identity-management/pim-getting-started.md) | Fully supported with Azure AD premium P2 licensing | Azure AD B2C tenants do not support Privileged Identity Management. |
+| [Entitlement Management](../active-directory/governance/entitlement-management-overview.md) | Fully supported with Azure AD premium P2 licensing | Azure AD B2C tenants do not support Entitlement Management. |
+| [Access Reviews](../active-directory/governance/access-reviews-overview.md) | Fully supported with Azure AD premium P2 licensing | Azure AD B2C tenants do not support Access Reviews. |
+| [Lifecycle Workflows](../active-directory/governance/what-are-lifecycle-workflows.md) | Fully supported with Azure AD premium P2 licensing | Azure AD B2C tenants do not support Lifecycle Workflows. |
+| [HR App Provisioning](../active-directory/governance/what-is-provisioning.md) | Fully supported with Azure AD premium P1 licensing | Azure AD B2C tenants do not support HR app provisioning. |
+| [Premium P1](https://azure.microsoft.com/pricing/details/active-directory) | Fully supported for Azure AD premium P1 features. For example, [Password Protection](../active-directory/authentication/concept-password-ban-bad.md), [Hybrid Identities](../active-directory/hybrid/whatis-hybrid-
```

Comment:
I feel the Azure AD Premium P1 vs. Azure AD B2C Premium P1 discussion should be a separate section that talks about licenses vs. pricing-based feature tiers.